app/main.py

```diff
@@ -1,8 +1,41 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(
+            self,
+            name: str,
+            age: int
+    ) -> None:
+        self.name = name
+        self.age = age
+        self.change_people(self)
+
+    @classmethod
+    def change_people(cls, self) -> None:
+        cls.people[self.name] = self
```

Comment:
The flake8 is arguing for absense of annotation for `self`.  Don't use it for naming your own arguments.
`self` is a first argument for instance methods and shouldn't be reused for other purposes.

No need to even move this code to a function, you can use it directly. This way you'll fix this flake8 error